configuration/hdmi-config.md

```diff
@@ -2,7 +2,7 @@
 
 In the vast majority of cases, simply plugging your HDMI-equipped monitor into the Raspberry Pi using a standard HDMI cable will automatically lead to the Pi using the best resolution the monitor supports. On the Raspberry Pi 4 there are two micro HDMI ports, so you will need either one or two micro-HDMI-to-full-size-HDMI leads or adapters, depending on how many displays you wish to attach. You should connect any HDMI leads before turning on the Raspberry Pi.
```

Comment:
Not directly related to this PR, but is it worth squeezing a sentence in here about the Raspberry Pi Zero using a mini HDMI port?  WDYT @JamesH65 ?

Reply:
Good point - we call out the Pi4, we should also call out the Zero.

Reply:
I would go the other way - if someone is buying a Pi it should be obvious if there is an adapter needed, from whatever webpage they bought it from. That way the docs stay simple.

Reply:
Although maybe it was me that put the bit about the adaptor for the Pi 4 in to start with lol.

Reply:
Already added (and merged) in #1376 :wink:

Reply:
I know.

Reply:
https://www.youtube.com/watch?v=4LaYn0541ro